### PR TITLE
fix: plot API annotations

### PR DESCRIPTION
**Pull request by taro-yu**
## Description

This PR fixs plot API annotations about TimeSeriesTypes.

## Related links

https://tier4.atlassian.net/browse/RT2-1203
## Pre-review checklist for the PR author

- [x] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [x] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR has been properly tested.
- [ ] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] (Optional) The PR has been properly tested with CARET_report verification.
- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: src/caret_analyze/plot/plot_facade.py [hunk 1]
```diff
@@ -103,15 +102,14 @@ def create_response_time_stacked_bar_plot(
 
     @staticmethod
     def create_period_timeseries_plot(
-        *target_objects: TimeSeriesTypes
+        *target_objects: CallbackBase | Communication
```
Comment (iwatake2222) [inline src/caret_analyze/plot/plot_facade.py:105]: This API should accept `Publisher`, `Subscription` as well.
When `Publisher` is set: This API outputs the period of the published topic
When `Subscription` is set: This API outputs the period of the subscription callback

File: src/caret_analyze/plot/plot_facade.py [hunk 2]
```diff
@@ -128,15 +126,14 @@ def create_period_timeseries_plot(
 
     @staticmethod
     def create_frequency_timeseries_plot(
-        *target_objects: TimeSeriesTypes
+        *target_objects: CallbackBase | Communication
```
Comment (iwatake2222) [inline src/caret_analyze/plot/plot_facade.py:129]: This API should accept `Publisher`, `Subscription` as well.
When `Publisher` is set: This API outputs the freq of the published topic
When `Subscription` is set: This API outputs the freq of the subscription callback

**Review by iwatake2222 — CHANGES_REQUESTED** [review]

File: src/caret_analyze/plot/plot_facade.py (hunk 2 above)
Comment (taro-yu) [reply src/caret_analyze/plot/plot_facade.py]: fixed in https://github.com/tier4/caret_analyze/pull/463/commits/830035f931d3315a54f495a053d4962bc17032f6

File: src/caret_analyze/plot/plot_facade.py (hunk 1 above)
Comment (taro-yu) [reply src/caret_analyze/plot/plot_facade.py]: fixed in https://github.com/tier4/caret_analyze/commit/830035f931d3315a54f495a053d4962bc17032f6

**Review by iwatake2222 — DISMISSED** [review]
Thank you for fixing it!

**Review by rokamu623 — CHANGES_REQUESTED** [review]
@taro-yu @iwatake2222 
The following changes seem necessary:

- create_frequency_histogram_plot
- create_period_histogram_plot
  - Add `Publisher` and `Subscription` annotations.

**Comment by taro-yu** [comment]
> @taro-yu @iwatake2222 The following changes seem necessary:
> 
> * create_frequency_histogram_plot
> * create_period_histogram_plot
>   
>   * Add `Publisher` and `Subscription` annotations.

fixed in https://github.com/tier4/caret_analyze/pull/463/commits/0ea9e86aa53b1e6a1d7b1d90a6e6d361fae7c1bb

**Review by rokamu623 — APPROVED** [review]
LGTM!
image

**Review by xygyo77 — COMMENTED** [review]
image

LGTM

**Review by xygyo77 — COMMENTED** [review]
LGTM

**Comment by taro-yu** [comment]
> LGTM

Thank you for your review.

**Pull request merged**